client/gutenberg/extensions/business-hours/components/day-save.jsx

```diff
@@ -6,11 +6,6 @@ import { date } from '@wordpress/date';
 import { isEmpty } from 'lodash';
 import { sprintf } from '@wordpress/i18n';
 
-/**
- * Internal dependencies
- */
-import { _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
-
 class DaySave extends Component {
```

Comment:
`DaySave` sounds weird as component name, are we using this `ComponentSave` notation elsewhere?

Reply:
no we don't. i'm not sure what else to call it

Reply:
We use similar notation here: https://github.com/Automattic/wp-calypso/tree/9073dcc3c49d2afadf602e042dd454779039ede1/client/gutenberg/extensions/tiled-gallery/gallery-image

I think important part is to call the other one `DayEdit` if there is `DaySave`

`DateRow` could be alternative for "Day"